### PR TITLE
add suites shedyamls

### PR DESCRIPTION
**Pull request by martenson**
includes fixes for @nsoranzo in the previous PR

Suites are generally troublesome because here they are not tied to the exact revisions of tools. I will go ahead and do some testing with this.

### CODE REVIEW

**Comment by martenson** [comment]
We might want to consider reverting https://github.com/galaxyproject/tools-devteam/commit/c2c801a6dd10a500b7e13d13f5f131e384bdad48

the way suites are versioned does not play well with the lack of versions in the actual repo dependencies.xml

**Comment by bgruening** [comment]
@martenson the revision should be filled during upload? Isn't this the case?

**Comment by martenson** [comment]
@bgruening it will fill the latest which breaks the versioning of suites

**Comment by bgruening** [comment]
True ... but suites without a version number are supposed to install the latest versions?
Sorry, seems that I messed this up.

**Comment by martenson** [comment]
I will revert part of #30 in a separate PR. Please review this one independently.

**Comment by martenson** [comment]
created PR for it here: https://github.com/galaxyproject/tools-devteam/pull/269

**Comment by martenson** [comment]
will merge today unless objections arise

File: suites/suite_gatk_1_4/.shed.yml [hunk 1]
```diff
@@ -1,4 +1,14 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/suite_gatk_1_4
-owner: devteam
+categories:
+- Sequence Analysis
```
Comment (nsoranzo) [inline suites/suite_gatk_1_4/.shed.yml]: s/Sequence/Variant/

**Pull request merged**